tests/testthat/test-renderer2-interactive-legends.R

```diff
@@ -53,7 +53,7 @@ test_that("clicking sepal.width legend does nothing", {
 })
 
 test_that("clicking Sepal point doesn't affect sepal plot", {
-  clickID("51")
+  clickElementById("51")
```

Comment:
@lazycipher instead of changing all clickID to clickElementById, can you please just leave all the clickID in the test-*.R files, and change the definition of clickID?

Reply:
Sure thing, I'll change that! I was experimenting something and forgot about that!